### ISSUE TITLE
Touchscreen input not working

### ISSUE THREAD

**Issue by AelarAmakiir** [ahezard/nds-bootstrap#856]
### Your console

Tell us which console you are using:

- [ ]  DS Phat/Lite
- [ ]  DSi (XL)
- [ ]  Old 3DS/2DS (XL)
- [x]  New 3DS/2DS (XL)

### CFW and configuration

Tell us which CFW do you have:

DSi:

- [ ]  (No CFW) DSiWarehax (such as Flipnote Lenny, Sudokuhax, etc.)
- [ ]  (No CFW) Unlaunch only _[please specify version]_
- [ ]  Unlaunch+HiyaCFW _[please specify versions of both]_

3DS/2DS:

- [x]  Luma3DS
- [ ]  Rei-Six
- [ ]  Other/Older CFW [specify]

### nds-bootstrap version

Tell us which versions you are using:

- [x]  Release _[v0.28.1]_
- [ ]  Nightly _[  ]_

### Your device

Tell us which card you are using:

- [x]  SD Card
- [ ]  (Slot-1) Flashcard

### Expected behaviour

When launching games, the touchscreen should work

### Actual behaviour

Nothing happens when the touch screen is used

### Steps to reproduce

1. Launch any DS game from the SD card
2. Wait for it to load
3. Attempt to use the touchscreen

### Other notes

Tested with LoZ: Phantom Hourglass and Brain Age (where the problem was first noticed) with both the release and nightly branches  
Almost certainly an nds-bootstrap problem and not TWiLight Menu as the touchscreen works just fine in TLM  
I'm using the Acekard theme, if that matters at all

**Comment by RocketRobz** [comment]
Try re-installing the latest version of nds-bootstrap.

**Comment by AelarAmakiir** [comment]
Still not working.
Another thing that might be good to know (after a bit more testing) is that tapping the screen will still open the main menu in some games (like Phantom Hourglass), but won't let me open save files. Also, in Pokemon Mystery Dungeon: Explorers of Sky, the player moves towards the top left corner of the screen, but only when the screen is presses.
tl;dr the screen still works but doesn't seem to have a location for touch points

**Comment by RocketRobz** [comment]
Try reinstalling the TWLMenu++ CIAs then.

**Comment by AelarAmakiir** [comment]
I tried re-installing, nothing seems to have changed. I'll try some other things, but I'm not sure if any of them will work.

**Comment by RocketRobz** [comment]
Send the MD5 of `sd:/_nds/nds-bootstrap-release.nds`, using http://onlinemd5.com

**Comment by AelarAmakiir** [comment]
nds-bootstrap-release.nds md5: `BB9DA71DF4A20CC44A1A0239A1711A4C`

**Comment by RocketRobz** [comment]
That's correct.

Are you running the games in DS mode, or forced DSi mode?

**Comment by AelarAmakiir** [comment]
DS mode

**Comment by RocketRobz** [comment]
Let's bring this over to the Discord server.
Hopefully, others can help.
